config/docker/fragment/kernelci.jinja2

```diff
@@ -12,7 +12,7 @@ RUN git clone --depth=1 $core_url /tmp/kernelci-core
 WORKDIR /tmp/kernelci-core
 RUN git fetch origin $core_rev
 RUN git checkout FETCH_HEAD
-RUN python3 -m pip install .
+RUN python3 -m pip install .[dev]
```

Comment:
```suggestion
RUN python3 -m pip install '.[dev]'
```

I'd rather be on the safe side and ensure the brackets are not interpreted by the shell.

Reply:
Thanks. Fixed the command.